canton/BUILD.bazel

```diff
@@ -55,17 +55,10 @@ java_import(
 
 genrule(
     name = "canton-lib-ee",
-    # We don't use the else branch but we need a resolvable value
-    srcs = [":lib/canton.jar"] if canton["local"] else ["@canton//:jar"],
     outs = ["canton-lib-ee.jar"],
     cmd = """
 set -euo pipefail
 
-if [ "{local}" = "true" ]; then
-    cp $(location {src}) $@
-    exit 0
-fi
-
```

Comment:
@garyverhaegen-da There is an issue here. We cannot use the arbitrary sha for `canton-ee`.
I think using always the snapshot for EE is good enough.

Reply:
@remyhaemmerle-da Windows is now failing with `cp: target 'bazel-out/x64_windows-opt/bin/canton/canton-lib-ee.jar' is not a directory` - are these changes also needed in a ps1 file?

Reply:
@remyhaemmerle-da I'm not sure I understand what you're trying to say here. Can you elaborate? What is preventing you fromusing the arbitrary sha for `canton-ee`?

Reply:
See these errors
https://dev.azure.com/digitalasset/daml/_build/results?buildId=145218&view=logs&j=52c0dd40-3465-5e8e-2f7c-ae591b3bddfa&t=199b63e7-8a1c-52a3-a4d5-f2bcbcf31b17&l=249